src/js/defaults.js

```diff
@@ -0,0 +1,18 @@
+import {validation as v} from './utils';
+
+let chartConfiguration = [{"label": "Value", "formatting": ",.3d"}];
+
+function checkAny(x, defaultVal) {
+    if (v.isMissingData(x))
+        return defaultVal;
+    return x;
+}
+
+
+export const defaults = {
+    checkAny
```

Comment:
I would move the `checkAny` method to utils and don't add a `defaults` export here.

Reply:
probably - I was on the fence about it - but I suppose that it could be used out of the context of defaults anyway